oeps/oep-1.rst

```diff
@@ -0,0 +1,266 @@
+are described below. All other rows are required.
+
+
++---------------+-------------------------------------------+
+| OEP           | <link to OEP>                             |
++---------------+-------------------------------------------+
+| Title         | <OEP title>                               |
++---------------+-------------------------------------------+
+| Last-Modified | <date string>                             |
++---------------+-------------------------------------------+
+| Author        | <list of authors' real names and          |
+|               | optionally, email addrs>                  |
++---------------+-------------------------------------------+
+| Arbiter       | <OEP czar's real name>                    |
++---------------+-------------------------------------------+
+| Status        | <Draft | Accepted | Deferred |            |
+|               | Rejected | Withdrawn | Final |            |
+|               | Replaced>                                 |
++---------------+-------------------------------------------+
+| Type          | <Architecture | Best Practice |           |
+|               | Process>                                  |
++---------------+-------------------------------------------+
+|  Created      | <date created on, in yyyy-mm-dd format>   |
++---------------+-------------------------------------------+
+| `Replaces`    | <OEP number>                              |
++---------------+-------------------------------------------+
+| `Replaced-By` | <OEP number>                              |
++---------------+-------------------------------------------+
+
+The Author header lists the names, and optionally the email addresses of
+all the authors/owners of the PEP. The format of the Author header value must be
+``Random J. User <address@dom.ain>`` if the email address is included, and just
+``Random J. User`` if the address is not given. If there are multiple authors,
+they should but put in a comma separated list.
+
+The Arbiter field is used to record who the final decision to approve or
+reject an OEP rests with.
+
+The Type header specifies the type of OEP: Architecture, Best Practice or Process.
+
+The Created header records the date that the OEP was assigned a number. It should be
+in yyyy-mm-dd format, e.g. 2016-04-21.
+
+OEPs may also have a Replaced-By header indicating that a OEP has been rendered
+obsolete by a later document; the value is the number of the OEP that replaces
+the current document. The newer OEP must have a Replaces header containing the
+number of the OEP that it rendered obsolete.
+
+Auxiliary Files
+---------------
+
+OEPs may include auxiliary files such as diagrams. Such files must be named put in the
+directory oep-XXXX/, where "XXXX" is the OEP number.
+
+Reporting OEP Bugs, or Submitting OEP Updates
+---------------------------------------------
+Bugs and updates to an OEP can be submitted as comments on the OEP PR, or by
```

Comment:
Do they need to be published someplace other than on github as comments attached to the pr?